api/overview/azure/latest/storage.md

```diff
@@ -70,6 +70,25 @@ Use the following version 25.x.x library to work with the Azure Storage resource
 
 This section details known issues for the Azure Storage client libraries for .NET.
 
+### Client-side encryption bug when using `BlobClient.UpdateClientSideEncryptionKey` with an unauthenticated symmetric encryption algorithm
+
+#### Issue description
+
+A bug was found in `BlobClient.UpdateClientSideEncryptionKey` that affects customers who are using an unauthenticated symmetric encryption algorithm. If a customer using an [impacted version](#issue-details) of **Azure.Storage.Blobs** attempts to rotate an unauthenticated symmetric encryption key using `BlobClient.UpdateClientSideEncryptionKey`, the key will not rotate correctly and decryption with the SDK might not be possible without additional steps. For remediation guidance, see [Recommended steps](#recommended-steps).
+
+This bug does *not* affect customers who use authenticated symmetric keys or asymmetric keys. Asymmetric keys are used by Azure Key Vault and are not affected by this bug. Also, the async API `BlobClient.UpdateClientSideEncryptionKeyAsync` is *not* affected by this bug.
```

Comment:
While they key itself is symmetric or asymmetric, it's the key-wrapping algorithm that is authenticated or not. We should probably be very specific in how we refer to these things.